source/_templates/registrations/linux/start_agent.rst

```diff
@@ -0,0 +1,24 @@
+.. Copyright (C) 2020 Wazuh, Inc.
+
+.. tabs::
+
+ .. group-tab:: Using Systemd
+
+  .. code-block:: console
+
+   # systemctl restart wazuh-agent
+
+ .. group-tab:: SysV Init
+
+  .. code-block:: console
+
+   # service wazuh-agent restart
+
+ .. group-tab:: For Unix host
```

Comment:
I don't understand this template. Start agent or restart agent?

Reply:
Anyway, the tabs titles must be: Systemd, SysV init, Other Unix based OS

Reply:
Done